src/test/java/io/vertx/mqtt/test/server/MqttServerClientCertSslTest.java

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright 2016 Red Hat Inc.
```

Comment:
Now it was just a copy -- should it have something like:
"Copyright © 2017 The Eclipse Foundation. All Rights Reserved."

@ppatierno can probably comment on this one?